src/dinit.cc

```diff
@@ -486,6 +587,11 @@ int dinit_main(int argc, char **argv)
         }
     }
 
+#if SUPPORT_SELINUX
+    // error exit if we are PID 1 and fail to load the selinux policy and transition
+    if (am_system_mgr && am_system_init && opts.load_selinux_policy && !selinux_transition(argv[0])) return 1;
```

Comment:
I recommend updating the comment here to explain why selinux_transition(argv[0]) is called here and why dinit should terminate when the if condition fails.

Reply:
Done